dev/tasks/r/github.linux.cran.yml

```diff
@@ -21,20 +21,21 @@
 
 jobs:
   as-cran:
-    name: "rhub/{{ MATRIX }}"
```

Comment:
Can we remove this?

https://github.com/apache/arrow/blob/12cdaaafb7a5ba39db81ba6d7e565b8e7a24117e/dev/tasks/tasks.yml#L1157-L1158

Reply:
Ah, yes, of course! I was wondering how that seemingly special `MATRIX` got the value it did! But we don't need it at the tasks.yml level at all. Thanks